docs/deployment/gardenlet_api_access.md

```diff
@@ -54,6 +54,7 @@ Today, the following rules are implemented:
 | `ControllerRegistration`    | `get`, `list`, `watch`                                          | `ControllerRegistration` -> `ControllerInstallation` -> `Seed`                                                                | Allow `get`, `list`, `watch` requests for all `ControllerRegistration`s.                                                                                                                                                                                                         |
 | `ControllerDeployment`      | `get`                                                           | `ControllerDeployment` -> `ControllerInstallation` -> `Seed`                                                                  | Allow `get` requests for `ControllerDeployments`s referenced by `ControllerInstallation`s assigned to the `gardenlet`'s `Seed`.                                                                                                                                                  |
 | `ControllerInstallation`    | `get`, `list`, `watch`, `update`, `patch`                       | `ControllerInstallation` -> `Seed`                                                                                            | Allow `get`, `list`, `watch` requests for all `ControllerInstallation`s. Allow only `update`, `patch` requests for `ControllerInstallation`s assigned to the `gardenlet`'s `Seed`.                                                                                               |
+| `CredentialsBinding`        | `get`                                                           | `CredentialsBinding` -> `Shoot` -> `Seed`                                                                                     | Allow only `get` requests for `CredentialsBinding`s referenced by `Shoot`s that are assigned to the `gardenlet`'s `Seed`.                                                                                                                                                        |
```

Comment:
This should be reflected in `Secret`s as well, see  https://github.com/gardener/gardener/blob/17d7d544baf4e5da64a5e46f6a137a42acee1b63/docs/deployment/gardenlet_api_access.md?plain=1#L66.